webapp/channels/src/components/admin_console/license_settings/enterprise_edition/enterprise_edition_right_panel.tsx

```diff
@@ -19,17 +19,65 @@ export interface EnterpriseEditionProps {
 const EnterpriseEditionRightPanel = ({
     isTrialLicense,
     license,
-
 }: EnterpriseEditionProps) => {
+    const intl = useIntl();
```

Comment:
nit:   if destructure like ` const {formatMessage} = useIntl();` you avoid having to do `intl.` several times. Same in the `starter_right_panel` file